OpenRA.Mods.Common/Widgets/ExponentialSliderWidget.cs

```diff
@@ -0,0 +1,52 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Drawing;
+using OpenRA.Graphics;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Widgets
+{
+	public class ExponentialSliderWidget : SliderWidget
+	{
+		public double ExpA = 1.0e-3;
+		public double ExpB = 6.908;
```

Comment:
It would be nice to have a comment here with a link to https://www.dr-lex.be/info-stuff/volumecontrols.html that explains where those numbers come from.